src/wpt/BUILD.bazel

```diff
@@ -66,6 +66,7 @@ wpt_test(
 
 wpt_test(
     name = "dom/events",
+    compat_flags = ["pedantic_wpt"],
```

Comment:
Should we maybe put this on for every wpt test? That'd be in `build/wpt_test.bzl`.

Reply:
I think eventually yes, but forcing it to be explicit on each one will likely help us find issues. When we've caught up, then probably putting it on all of them makes sense.